compute_block_size() returns always 1.
initializing a new partition table fails because the disk layout is wrong.

 file_media.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

Index: pdisk/file_media.c
===================================================================
--- pdisk.orig/file_media.c
+++ pdisk/file_media.c
@@ -90,7 +90,7 @@ struct file_media_iterator {
  * Global Constants
  */
 int potential_block_sizes[] = {
-    1, 512, 1024, 2048,
+    512, 1024, 2048,
     0
 };
 
